Sources/RedesignBezierSwift/Foundation/Typography/FontFamily.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  File.swift
+//  
+//
+//  Created by Tom on 2/7/24.
+//
+
+import UIKit.UIFont
+
+enum FontFamily {
+  enum Pretendard {
+    static let bold = FontConvertible(name: "Pretendard-Bold", family: "Pretendard", path: "Pretendard-Bold.otf")
+    static let regular = FontConvertible(name: "Pretendard-Regular", family: "Pretendard", path: "Pretendard-Regular.otf")
+    static let semiBold = FontConvertible(name: "Pretendard-SemiBold", family: "Pretendard", path: "Pretendard-SemiBold.otf")
+    static let all: [FontConvertible] = [bold, regular, semiBold]
+  }
+  static let allCustomFonts: [FontConvertible] = [Pretendard.all].flatMap { $0 }
```

Comment:
```suggestion
  static let allCustomFonts: [FontConvertible] = Pretendard.all
```
flatMap 하지 않고 바로 all을 가져다 써도 될것 같아요